### DISCUSSION TITLE
Stage 1: Basic Repo, Project Board and Wiki Setup

### DISCUSSION THREAD

**Discussion by sanket1211996** [airavata-courses/SuperNova#8] [Show and tell]
#4  I have created wiki for microservice A. Richa will be updating and anding pages related to this microservice under this wiki link. 

https://github.com/airavata-courses/SuperNova/wiki/Microservice-A:-User-APi

**Comment by sanket1211996** [comment]
#2 I have created wiki for gateway A. @sumedhsalvi  will be updating and anding pages related to this gateway under this wiki link.

https://github.com/airavata-courses/SuperNova/wiki/Gateway:-FastApi

**Reply by sumedhsalvi** [reply]
Sure, will do!

**Reply by sumedhsalvi** [reply]
I have checked in the basic repo and raised PR. #2

**Reply by sanket1211996** [reply]
@sumedhsalvi  The basic get api of hello world seems fine for now. I have approved and merged the code.

**Reply by sumedhsalvi** [reply]
Okay, Thanks!

**Comment by sanket1211996** [comment]
#3   Also, I have created UI angular wiki. I will be adding me pages under that wiki page.

https://github.com/airavata-courses/SuperNova/wiki/User-Interface:-Angular

**Reply by sanket1211996** [reply]
I have checked in basic repo.

**Reply by sanket1211996** [reply]
Do move this issue to Done.

**Comment by sanket1211996** [comment]
Hi @richajha012 , whats the update of this issue #4 .

**Reply by richajha012** [reply]
I have done with basic repo and raised PR for this issue #4

**Reply by richajha012** [reply]
Let's review and merge.

**Reply by sanket1211996** [reply]
This seems good for now. Merging this PR. make new feature branch for further changes. Good Work

**Reply by sanket1211996** [reply]
@richajha012  this PR is merged. Move issue to done and close the issue with comment.
